airbyte-integrations/bases/airbyte-protocol/.gitignore

```diff
@@ -0,0 +1 @@
+airbyte_protocol/models/yaml
```

Comment:
There isn't anything in this location. This `.gitignore` file can be deleted.

Reply:
thanks.